python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -608,6 +608,7 @@ def __init__(
         self._environment_vars = check.opt_mapping_param(
             environment_vars, "environment_vars", key_type=str, value_type=str
         )
+        self._environment_vars = self._environment_vars or None
```

Comment:
We can use `opt_nullable_mapping_param` instead of `opt_mapping_param` to avoid this line